### ISSUE TITLE
IfcShapeRepresentation.RepresentationType for IfcCurveSegment

### ISSUE THREAD

**Issue by czapplitec** [bSI-InfraRoom/IFC-Specification#115]
When IfcCurveSegment is used as geometry for IfcAlignmentSegment, it requires a proper RepresentationType. It seems "Curve", "Curve2D" and "Curve3D" could match, but they violate the Function IfcShapeRepresentationTypes in the schema. This function needs to be updated.

**Comment by SergejMuhic** [comment]
I would prefer having a new type "Curve segment" as IfcCurveSegment is not a curve in its own right.

**Comment by czapplitec** [comment]
I agree. A new type "CurveSegment" is better. As far as I see, most current IFC files use "Curve2D", which violate the current schema. 
Either we agree on "CurveSegment" or "Curve2D", the Funtion IfcShapeRepresentationTypes needs to be extended.

**Comment by czapplitec** [comment]
Or just "Segment", to be used for IfcSegment

**Comment by SergejMuhic** [comment]
Yes, `Segment` would also be good. The documentation would then be something like:

"A segment of a composite curve."
